src/server/handlers/github.js

```diff
@@ -137,3 +149,35 @@ const getRequest = (owner, name, token, secret) => {
     }
   };
 };
+
+/*
+ * parse_link_header()
+ *
+ * Parse the Github Link HTTP header used for pageination
+ * http://developer.github.com/v3/#pagination
+ *
+ * Modified by kfenn to return page numbers instead of urls
```

Comment:
I feel that if we're working out page numbers anyway then we might as well present those rather than first/prev/next/last, but that can be in a later PR.